docs/concepts/service-catalog/index.md

```diff
@@ -0,0 +1,234 @@
+
+1. Listing the Managed Services available from a Service Broker.
+1. Provisioning a new instance of the Managed Service.
+1. Binding to the Managed Service, which returns the connection credentials.
+1. Mapping the connection credentials into the application.
+
+### Listing Managed Services
+
+First, a {% glossary_tooltip text="Cluster Operator" term_id="cluster-operator" %} must create a `ServiceBroker` resource within the `servicecatalog.k8s.io` group. This resource contains the URL and connection details necessary to access a Service Broker endpoint.
+
+This is an example of a `ServiceBroker` resource:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: ServiceBroker
+metadata:
+  name: cloud-broker
+spec:
+  # Points to the endpoint of a Service Broker. (This example is not a working URL.)
+  url:  https://servicebroker.somecloudprovider.com/v1alpha1/projects/service-catalog/brokers/default
+  # Describes the secret which contains the short-lived bearer token
+  authInfo:
+    bearer:
+      secretRef:
+        name: cloud-svc-account-secret
+        namespace: service-catalog
+```
+
+The following is a sequence diagram illustrating the steps involved in listing Managed Services available from a Service Broker:
+
+![List Services](/images/docs/service-catalog-list.svg){:height="80%" width="80%"}
+
+1. Once the `ServiceBroker` resource is added to Service Catalog, it triggers a *List Services* call to the external Service Broker.
+1. The Service Broker returns a list of available Managed Services, which is cached locally in a `ServiceClass` resource.
+1. A {% glossary_tooltip text="Cluster Operator" term_id="cluster-operator" %} can then get the list of available Managed Services using the following command:
+
+        kubectl get serviceclasses
+
+### Provisioning a new instance
+
+A {% glossary_tooltip text="Cluster Operator" term_id="cluster-operator" %} can initiate the provisioning of a new instance by creating a `ServiceInstance` resource. 
+
+This is an example of a `ServiceInstance` resource:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: ServiceInstance
+metadata:
+  name: cloud-queue-instance
+  namespace: cloud-apps
+spec:
+  # References one of the previously returned services
+  serviceClassName: queue
+  planName: queue-plan
+```
+
+The following sequence diagram illustrates the steps involved in provisioning a new instance of a Managed Service:
+
+![Provision a Service](/images/docs/service-catalog-provision.svg){:height="80%" width="80%"}
+
+1. When the `ServiceInstance` resource is created, Service Catalog initiates a *Provision Instance* call to the external Service Broker.
+1. The Service Broker creates a new instance of the Managed Service and returns an HTTP 200 response if the provisioning was successful.
```

Comment:
The provisioning happens in the background so 200 response doesn't necessarily means provisioning successful. So I would keep it to minimum.

Reply:
ack